options/locale/locale_en-US.ini

```diff
@@ -61,97 +61,97 @@ cancel = Cancel
+title = Initial Configuration
+docker_helper = If you run Gitea inside Docker, please read the <a target="_blank" rel="noopener" href="%s">documentation</a> before changing any settings.
+requite_db_desc = Gitea requires MySQL, PostgreSQL, MSSQL, SQLite3 or TiDB.
 db_title = Database Settings
 db_type = Database Type
 host = Host
-user = User
+user = Username
 password = Password
 db_name = Database Name
-db_helper = Please use the INNODB engine with utf8_general_ci charset for MySQL.
-ssl_mode = SSL Mode
+db_helper = Note to MySQL users: please use the InnoDB storage engine and the 'utf8_general_ci' character set.
+ssl_mode = SSL
 path = Path
-sqlite_helper = The file path to the SQLite3 or TiDB database. <br>Please use the absolute path when you start as service.
-err_empty_db_path = SQLite3 or TiDB database path cannot be empty.
-err_invalid_tidb_name = TiDB database name does not allow characters "." and "-".
-no_admin_and_disable_registration = You cannot disable registration without creating an admin account.
-err_empty_admin_password = Admin password cannot be empty.
-
-general_title = General Application Settings
-app_name = Application Name
-app_name_helper = You can put your organization name here.
+sqlite_helper = File path for the SQLite3 or TiDB database.<br>Enter an absolute path if you run Gitea as a service.
+err_empty_db_path = The SQLite3 or TiDB database path cannot be empty.
+err_invalid_tidb_name = The TiDB database name cannot contain the '.' and '-' characters.
+no_admin_and_disable_registration = You cannot disable user self-registration without creating an administrator account.
+err_empty_admin_password = The administrator password cannot be empty.
+
+general_title = General Settings
+app_name = Site Title
+app_name_helper = Enter your company name here.
 repo_path = Repository Root Path
-repo_path_helper = All remote Git repositories will be saved to this directory.
-lfs_path = LFS Root Path
-lfs_path_helper = Files stored with Git LFS will be stored in this directory. Leave empty to disable LFS.
-run_user = Run User
-run_user_helper = The user must have access to Repository Root Path and run Gitea.
-domain = Domain
-domain_helper = This affects SSH clone URLs.
-ssh_port = SSH Port
-ssh_port_helper = Port number which your SSH server is using. Leave it empty to disable.
-http_port = HTTP Port
-http_port_helper = Port number which application will listen on.
-app_url = Application URL
-app_url_helper = This affects HTTP/HTTPS clone URL and some email notifications.
+repo_path_helper = Remote Git repositories will be saved to this directory.
+lfs_path = Git LFS Root Path
+lfs_path_helper = Files tracked by Git LFS will be stored in this directory. Leave empty to disable.
+run_user = Run As Username
+run_user_helper = Enter the operating system username that runs Gitea. Note that this user must have access to the repository root path.
+domain = Server SSH Domain
```

Comment:
This should probably be better as `SSH Server Domain`

Reply:
I wanted to emphasize that this is related to the actual server (as a whole), instead of any built-in SSH server functionality. I'll switch it around for now.